documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -854,8 +906,8 @@ EOF
 <6> Specify the name of the `NetworkMap` CR.
 <7> Specify a storage mapping even if the VMs to be migrated are not assigned with disk images. The mapping can be empty in this case.
 <8> Specify the name of the `StorageMap` CR.
-<9> You can use either the `id` or the `name` parameter to specify the source VMs.
-<10> Specify the {rhv-short} VM UUID.
+<9> Specify the {rhv-short} VM UUID.
```

Comment:
in the CLI:
`You can use either the `id` or the `name` parameter to specify the source VMs
`

this is important to CLI users who don't have easy access to the VM UUID
forcing users to find the UUID may not be good experience, and we do allow it in the CLI

Reply:
Change reverted, as above.